.github/ISSUE_TEMPLATE/request-an-article.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Request an article
+about: Request a topic for a new article
+title: 'Article: '
+labels: 'article'
+assignees: ''
+
+---
+
+_Please replace the content with the relevant details._
+
+**Article title:** _e.g. “Multi-engine machine translation”_
+
+**Directory:** _e.g. “Applications”. If necessary, suggest a new directory._
+
+**Description:** _e.g. “Multi-engine machine translation is an approach for evaluating and picking a machine translation engine”._
```

Comment:
- Let's do it for an article that already exists (I'm not sure the description is accurate).
- This is more a definition.  Let's ask for a description and a definition.

> **Description:** *e.g. “Machine translation for Russian”*
> 
> **Definition:** *e.g. “**Russian** Russian was one of the first languages for which machine translation was researched and developed, and remains one of the best supported languages.”.*

Reply:
We use `*` not `_` for italics throughout, maybe here too?